compiler-core/src/ast.rs

```diff
@@ -417,6 +418,10 @@ impl<T> Import<T> {
         }
     }
 
+    pub(crate) fn compare(&self, other: &Self) -> Ordering {
```

Comment:
Could you remove this please? I think it's clearer what's happening to compare the names in the formatter.